python-sdk/nuscenes/nuscenes.py

```diff
@@ -737,7 +763,16 @@ def map_pointcloud_to_image(self,
         elif show_lidarseg_labels:
             assert pointsensor['sensor_modality'] == 'lidar', 'Error: Can only render lidarseg labels for lidar, ' \
                                                               'not %s!' % pointsensor['sensor_modality']
-            lidarseg_labels_filename = osp.join(self.nusc.dataroot, 'lidarseg', pointsensor_token + '_lidarseg.bin')
+
+            if show_lidarseg_preds:
+                sample_token = self.nusc.get('sample_data', pointsensor_token)['sample_token']
+                lidarseg_labels_filename = osp.join(show_lidarseg_preds, pointsensor_token + '.bin')
```

Comment:
Standardize file format to be the same as for GT

Reply:
This is no longer relevant as users now input the entire file path to the .bin file which contains their preds